recipes/monet/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "monet" %}
+{% set version = "2.1.5" %}
+
+package:
+  name: monet
+  version: {{ version }}
+
+source:
+  #git_url: https://github.com/noaa-oar-arl/MONET.git
+  #git_tag: develop
+  url: https://github.com/noaa-oar-arl/MONET/archive/v2.2.0.tar.gz
```

Comment:
Use the jinja templating variable for version in the url so it's easier for the bots to generate a PR when there is an update.

Reply:
they have been synced